README.md

```diff
@@ -37,11 +37,135 @@ Before you clone and run the project, it is advisable to go through the short tu
 Run the mobile project on desktop:
 
 ```
-./gradlew :DevoxxClient:run
+mvn -f DevoxxClientMobile javafx::run
 ```
 
+
+## Building
+
+## Common build steps
+
+Both Android and ios builds have a common requisite to build: uses graalvm.
+You will need to have gluon graalvm own distribution in order to generate a binary package to install or distribute.
+You can download latest graalvm distribution from [here](https://github.com/gluonhq/graal/releases/latest)
+This distro workds with java17 and graalvm 22.0.0.3 (note: gluon's distribution shows itself as 22.0.0.2 in logs)
+Once you have installed graalvm you need to inform where is graalvm home using GRAALVM_HOME environment variable.
+
+MacOS example:
+
+```
+export GRAALVM_HOME=/path/to/graalvm-svm-java17-darwin-gluon-22.0.0.3-Final/Contents/Home
+```
+
+### Android
+
+#### Prerequisites
+
+Android packages only can be built in linux operating system (or WSL2 in Windows, but not in Mac).
+
+In addition Android packages only can be built to architecture aarch64, not compatible with ARM mobiles.
+
+To build gluonfx packages first you need to install:
+
+- Gluonfx graalvm distribution (https://github.com/gluonhq/graal/releases/latest)
+- gcc (6+) and ld (2.26+) packages
+- Android SDK and optionally Android NDK
+- "platform-tools","platforms;android-30","build-tools;30.0.2","ndk-bundle","extras;android;m2repository","extras;google;m2repository" android packages.
+
+#### Build
+
+In file DevoxxClientMobile/pom.xml change the line
+
+```xml
```

Comment:
Most of my comments are opinionated, except this.
We should not requier manual steps. We don't need it current PR, but we should create a task (trello or GH issue here) to have maven profiles for this.

Reply:
i tried other forms to do this (profiles or maven parameters). Doesn't work